server/src/domain/shared-link/shared-link.core.ts

```diff
@@ -71,6 +71,7 @@ export class SharedLinkCore {
             isAllowUpload: link.allowUpload,
             isAllowDownload: link.allowDownload,
             isShowExif: link.showExif,
+            externalPath: user.externalPath,
```

Comment:
Why does shared link is related in this PR?

Reply:
I think this was accidental, me attempting to get externalPath everywhere the user was exported. Will remove this.